CMakeLists.txt

```diff
@@ -39,6 +39,9 @@ set(LIBHAT_SRC
     src/arch/arm/System.cpp)
 
 add_library(libhat STATIC ${LIBHAT_SRC})
+if(UNIX)
+    set_target_properties(libhat PROPERTIES POSITION_INDEPENDENT_CODE ON)
```

Comment:
I imagine this should be set for the libhat_c target too?

Reply:
That property is automatically set for SHARED libraries

Reply:
There's an option to create the target as STATIC

https://github.com/BasedInc/libhat/blob/f4755aea0987e77c0f6f496c49eb9cd08d5f5a06/CMakeLists.txt#L58-L59

Reply:
Honestly now that I think about it I'm not sure why the compiler complained for libhat given that it is statically linked. There shouldn't be a downside to enabling it for libhat_c as well, as thats enabled by default on Windows anyways